Microsoft.Toolkit.Uwp.Services/Services/Twitter/Tweet.cs

```diff
@@ -44,22 +38,91 @@ public string Text
         }
 
         /// <summary>
-        /// Gets or sets the extended mode.
+        /// Gets or sets text of the tweet (280 characters).
+        /// </summary>
+        [JsonProperty("full_text")]
+        private string FullText { get; set; }
+
+        /// <summary>
+        /// Gets or sets display text range (indexes of tweet text without RT and leading user mentions)
+        /// </summary>
+        [JsonProperty("display_text_range")]
+        public int[] DisplayTextRange { get; set; }
+
+        /// <summary>
+        /// Gets or sets truncated flag (true when tweet is longer than 140 characters)
+        /// This entity may be deprecated - it never seems to be set to true.
+        /// </summary>
+        [JsonProperty("truncated")]
+        public bool Truncated { get; set; }
+
+        /// <summary>
+        /// Gets or sets attached content of the tweet
+        /// </summary>
+        [JsonProperty("entities")]
+        public TwitterEntities Entities { get; set; }
+
+        /// <summary>
+        /// Gets or sets extended attached content of the tweet
+        /// </summary>
+        [JsonProperty("extended_entities")]
+        public TwitterExtendedEntities ExtendedEntities { get; set; }
+
+        /// <summary>
+        /// Gets or sets full 280-character tweet text.
         /// </summary>
+        [Obsolete("This member has been deprecated by Twitter. Use Text instead.")]
         [JsonProperty("extended_tweet")]
         public TwitterExtended Extended { get; set; }
 
+        /// <summary>
+        /// Gets or sets tweet source (client or website used)
+        /// </summary>
+        [JsonProperty("source")]
+        public string Source { get; set; }
+
+        /// <summary>
+        /// Gets or sets in_reply_to_screen_name
+        /// </summary>
+        [JsonProperty("in_reply_to_screen_name")]
+        public string InReplyToScreenName { get; set; }
+
+        /// <summary>
+        /// Gets or sets in_reply_to_status_id_str
+        /// </summary>
+        [JsonProperty("in_reply_to_status_id_str")]
+        public string InReplyToStatusId { get; set; }
+
+        /// <summary>
+        /// Gets or sets in_reply_to_user_id_str
+        /// </summary>
+        [JsonProperty("in_reply_to_user_id_str")]
+        public string InReplyToUserId { get; set; }
+
         /// <summary>
         /// Gets or sets user who posted the status.
         /// </summary>
         [JsonProperty("user")]
         public TwitterUser User { get; set; }
 
         /// <summary>
-        /// Gets or sets attached content of the tweet
+        /// Gets or sets geo data (latitude and longitude).
         /// </summary>
-        [JsonProperty("entities")]
-        public TwitterEntities Entities { get; set; }
+        [Obsolete("This API has been deprecated by Twitter. Use Coordinates and Place instead.")]
+        [JsonProperty("geo")]
+        public TwitterGeoData GeoData { get; set; }
```

Comment:
Put these obsolete properties in the same area they were. This helps with looking at diff files between the versions to understand what changed. The easiest way is to place them under the property they were under before.